helm-chart/templates/secret.yaml

```diff
@@ -10,7 +10,6 @@ data:
   OPERATOR_KEY_MAIN: {{ .Values.config.OPERATOR_KEY_MAIN | b64enc }}
   OPERATOR_ID_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_ID_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
   OPERATOR_KEY_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_KEY_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
```

Comment:
Why are these operator properties not base64 encoded and are just escaped? Better to use `stringData` instead of `data` to avoid all the extra base64 conversion anyway.

Reply:
Agreed. @rustyShacklefurd feel free to open a separate PR that addresses this properly.
This satisfies the work around

Reply:
Added #301 to capture this

Reply:
good suggestion.  I will put this change in new branch and get it PR'd